week-2-wide-perspective/w02e02-data-manipulation/src/context/fake.context.js

```diff
@@ -0,0 +1,11 @@
+
+import { getOrders } from "../services/http.service.js"
+// import { State } from "../observers/ordersObserver.js"
+
+export const allAvailableOrders = await getOrders()
+// // export const ordersAmount = allAvailableOrders.length
+// // export const totalIncome = allAvailableOrders.reduce((total, item) => total + item.sale, 0).toFixed(2)
+
+// // export const bestSale = allAvailableOrders.sort((a, b) => b.sale - a.sale)[0].sale
+
+
```

Comment:
Nie za bardzo rozumiem czemu to siedzi w `context` prawdopodobnie tutaj zostało to zrefaktoryzowane (bo początkowo było np. `in memory` - ale nazwa pozostała :)